nbs/examples/Large_Datasets_Tutorial.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>If you update utilsforecast to 0.2.0 this can now be done like this:</p>

<pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;&nbsp; evaluate(
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; predictions.merge(target.drop(["trend", "y_[lag12]"], axis=1), on=['id_col', 'ds']),
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; metrics=[mae, rmse, smape],
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; id_col='id_col',
        agg_fn='mean',
&nbsp;&nbsp;&nbsp; )
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>